index.html

```diff
@@ -0,0 +1,169 @@
+      </a>
+    </div>
+    <svg class="hamb__close" width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+      <path fill-rule="evenodd" clip-rule="evenodd"
+        d="M5.29289 5.29289C5.68342 4.90237 6.31658 4.90237 6.70711 5.29289L12 10.5858L17.2929 5.29289C17.6834 4.90237 18.3166 4.90237 18.7071 5.29289C19.0976 5.68342 19.0976 6.31658 18.7071 6.70711L13.4142 12L18.7071 17.2929C19.0976 17.6834 19.0976 18.3166 18.7071 18.7071C18.3166 19.0976 17.6834 19.0976 17.2929 18.7071L12 13.4142L6.70711 18.7071C6.31658 19.0976 5.68342 19.0976 5.29289 18.7071C4.90237 18.3166 4.90237 17.6834 5.29289 17.2929L10.5858 12L5.29289 6.70711C4.90237 6.31658 4.90237 5.68342 5.29289 5.29289Z"
+        fill="#979493" />
+    </svg>
+  </section>
+  <!-- Hamburguer menu elements-->
+
+
+
+
+    <section class="first__section">
+      <div class="rocket__global">
+        <h3>"Hello Rocket world!"</h3>
+        <h1>ROCKET GLOBAL SUMMIT 2023</h1>
+      </div>
+      <p>A joyful celebration believing in the value of engineering, creating a positive change with people from over 80
+        countries is taking place in October, in Colombia.</p>
+      <div>
+        <h2 class="rocket__global__date">2023.10.17(THU) ~ 20(FRI)</h2>
+        <h3 class="rocket__global__place">Corferias Main Room, National University and more
+        </h3>
+      </div>
+
+    </section>
+  </div>
+  <section id="program" class="main__program">
+    <h1>Main Program</h1>
+    <div class="main__line"></div><!-- orange underline-->
+    <section class="main__program__cards">
+      <div class="card">
+        <img src="./images/lecture.png" alt="Lecture">
+        <h3>Lecture</h3>
+        <p>Listen to the speakers from various countries about the messages of sharing and opening.</p>
+      </div>
+      <div class="card">
+        <img src="./images/exhibition.png" alt="Exhibition">
+        <h3>Exhibition</h3>
+        <p>Appreciate various creations applying CC license of artists, organized from Art Center Nabi</p>
+      </div>
+      <div class="card">
+        <img src="./images/forum.png" alt="Forum">
+        <h3>Forum</h3>
+        <p>Have the time to share your thoughts and opinions with experts for each topic.</p>
+      </div>
+      <div class="card">
+        <img src="./images/workshop.png" alt="Workshop">
+        <h3>Workshop</h3>
+        <p>Try creating your own work using open source license rather than just watching at it.</p>
+      </div>
+      <div class="card">
+        <img src="./images/ignite.png" alt="Lecture">
+        <h3>Ignite</h3>
+        <p>Get opportunities to network with CC affiliates around the world, also after the summit.</p>
+      </div>
+    </section>
+    <button class="join__rocket desktop__hide" type="button">Join Rocket Global Summit 2023</button>
+    <a class="see__program hide desktop__show">SEE THE WHOLE PROGRAM</a>
+  </section>
+  <section id="speakers">
+    <h1>Featured Speakers</h2>
```

Comment:
- [ ] You open an `<h1>` and closed it with `</h2> `  . Please make sure to fix that errors that can affect your website readability and generate a bug :+1: